### DISCUSSION TITLE
Hide log from terminal

### DISCUSSION THREAD

**Discussion by marzuk-zarir** [yandeu/five-server#36] [Help / Questions] [answered]
How can I hide **[Five Server] change detected** log from terminal?

**Comment by yandeu** [comment, marked as answer]
Not sure. Have you tried [`logLevel`](https://github.com/yandeu/five-server/blob/e3af39ab4b4b987a62d2b2fe369d5018c445ae06/src/types.ts#L34)?

**Reply by marzuk-zarir** [reply]
Problem solved....thnx
